src/launch/cli/service/build/__init__.py

```diff
@@ -53,6 +53,11 @@
     default=False,
     help="(Optional) Skip cloning the application files. Will assume you're in a directory with the application files.",
 )
+@click.option(
```

Comment:
IMO we should not be providing a default value as aws. since this tool is designed to work with multiple cloud, we should force user to pass it than having a default value.

Reply:
Agreed!

I also took a look at how provider was being used and for the build step, we have logic that looks for AWS (fine). When I looked at how the publish command handled this, I found the logic it invoked treated npm as a provider, and npm really isn't a cloud provider to the extent of AWS or Azure, so it's not really clear to me what a provider means at that point.

Reply:
Default value is removed!! parameter is renamed to `--registry-type`